catalogs/catalog-jdbc-oceanbase/src/main/java/org/apache/gravitino/catalog/oceanbase/operation/OceanBaseTableOperations.java

```diff
@@ -217,7 +217,7 @@ public void alterTable(String databaseName, String tableName, TableChange... cha
         String sql = generateAlterTableSql(databaseName, tableName, change);
         if (StringUtils.isEmpty(sql)) {
           LOG.info("No changes to alter table {} from database {}", tableName, databaseName);
-          return;
+          continue;
```

Comment:
I'm generally okay with the change. @mchades, please also check if we intentionally return directly if the SQL string is empty.

Reply:
@pandeysambhi I'm not sure when `alterTableSql` will be empty. Could you please add tests to cover this scenario?